framework/Source/DirectionalShine.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  DirectionalShine.swift
+//
+//
+//  Created by Chiaote Ni on 2024/4/24.
+//
+
+import Foundation
+
+final class DirectionalShine: OperationGroup {
+
+    var rayLength: Float = 0.08 {
+        didSet {
+            firstDirectionalBlurEffect.length = rayLength
+            secondDirectionalBlurEffect.length = rayLength * 0.4
+        }
+    }
+    var degree: Float = 45.0 {
+        didSet {
+            firstDirectionalBlurEffect.degree = degree
+            secondDirectionalBlurEffect.degree = degree
+        }
+    }
+    var sparkleExposure: Float = 2 {
+        didSet {
+            exposureEffect.exposure = 2 + sparkleExposure
+        }
+    }
+
+    private let exposureEffect = ExposureAdjustment()
+    private let firstDirectionalBlurEffect = CBDirectionBlur()
+    private let secondDirectionalBlurEffect = CBDirectionBlur()
```

Comment:
Follow same name

either `DirectionBlur` or `DirectionalBlur`

Reply:
It should be DirectionalBlur, I'll rename them later on; thanks